src/cli/bin.ts

```diff
@@ -1,91 +1,26 @@
 import yargs from "yargs";
-import _capitalize from "lodash/capitalize";
-import _fromPairs from "lodash/fromPairs";
-import _mapValues from "lodash/mapValues";
-import fs from "fs-extra";
-import path from "path";
-import { ContentRecord, DataModel } from "@activeviam/activeui-sdk-5.1";
-import { getIndexedDataModel } from "@activeviam/data-model-5.1";
-import {
-  BehaviorOnError,
-  MigrationFunction,
-  OutcomeCounters,
-} from "../migration.types";
+import { BehaviorOnError } from "../migration.types";
 import { gte, coerce } from "semver";
-import { getMigrateDashboards } from "../getMigrateDashboards";
-import { getMigrateSavedWidgets } from "../getMigrateSavedWidgets";
-import { getMigrateSavedFilters } from "../getMigrateSavedFilters";
-import { migrate_43_to_50 } from "../4.3_to_5.0";
-import { migrate_50_to_51 } from "../5.0_to_5.1";
-import { getContent } from "../getContent";
+import { migrateContentServer } from "./scripts/migrateContentServer";
+import { migrateNotebook } from "./scripts/migrateNotebook";
+import { getFileType } from "./scripts/getFileType";
 
-const migrationSteps: {
+const AUIMigrationSteps: {
```

Comment:
The `AUIMigrationSteps` name is misleading, if I understand correctly, this maps the `from which version` to `to which version` the CLI can operate on. It would be simpler to inline it where its used.